iron-multi-selectable.html

```diff
@@ -61,11 +61,9 @@
         if (this.selectedValues) {
           this._toggleSelected(value);
         } else {
-          this.selectedValues = [value];
+          this.set('selectedValues',[value]);
```

Comment:
If there is a browser inconsistency in the way databinding behaves, this should be reported as an issue against Polymer core. I will create a test case to verify what you are saying.